src/test/ui/process/process-spawn-nonexistent.rs

```diff
@@ -1,6 +1,7 @@
 // run-pass
 // ignore-emscripten no processes
 // ignore-sgx no processes
+// ignore-fuchsia ErrorKind not translated
```

Comment:
Can you open an issue for this on this Github repo and mention it here? It seems like a bug in libstd's Fuchsia support.

Reply:
Done at https://github.com/rust-lang/rust/issues/102077